src/other/Table.js

```diff
@@ -96,5 +125,22 @@
     Table.prototype.click = function (d) {
     };
 
+    Table.prototype.testData = function(_) {
+        this.columns(["Lat", "Long", "Pin"])
+        this.data([
+            [ 37.665074, -122.384375, "green-dot.png" ],
+            [ 32.690680, -117.178540 ],
+            [ 39.709455, -104.969859 ],
+            [ 41.244123, -95.961610 ],
+            [ 32.688980, -117.192040 ],
+            [ 45.786490, -108.526600 ],
+            [ 45.796180, -108.535652 ],
+            [ 45.774320, -108.494370 ],
+            [ 45.777062, -108.549835, "red-dot.png" ]
+        ]);
+        
+        return this;
+    };
+
```

Comment:
If you want to re-factor code (or generally make changes wich have no affect on the logic), you should do it as a separate pull request - as I have no way seeing what changed in testData,,,